doc/modules/preprocessing.rst

```diff
@@ -423,3 +423,24 @@ values than observed values.
 
 :class:`Imputer` can be used in a Pipeline as a way to build a composite
 estimator that supports imputation. See :ref:`example_missing_values.py`
+
+
+Custom Transformers
+===================
+
+Often, you will want to convert an existing python function into a transformer
+to assist in data cleaning or processing. Users may implement a transformer from
```

Comment:
@mblondel would you still stick with the name then?
I agree on @vene's comment.